colossalai/inference/modeling/models/nopadding_baichuan.py

```diff
@@ -0,0 +1,181 @@
+# This code is adapted from huggingface baichuan model: hhttps://huggingface.co/baichuan-inc/Baichuan2-13B-Base/blob/main/modeling_baichuan.py
+from typing import Optional, Tuple
+
+import torch
+import torch.nn as nn
+
+from colossalai.inference.flash_decoding_utils import FDIntermTensors
+from colossalai.inference.modeling.models.nopadding_llama import llama_base_attn_forward
+from colossalai.kernel.kernel_loader import InferenceOpsLoader
+from colossalai.logging import get_dist_logger
+
+inference_ops = InferenceOpsLoader().load()
+
+logger = get_dist_logger(__name__)
+
+# def get_alibi_slopes(n_head):
+
+
+class NopadBaiChuanAttention(nn.Module):
```

Comment:
Exactly the same as `NopadLlamaAttention` in `nopadding_llama.py`. Why not just replace as llama layer in baichuan policy, then we don't need to put redundant layers in modeling

Reply:
Done